llvm/lib/Target/AMDGPU/GCNSubtarget.h

```diff
@@ -160,6 +160,7 @@ class GCNSubtarget final : public AMDGPUGenSubtargetInfo,
   bool HasAtomicFaddNoRtnInsts = false;
   bool HasAtomicBufferGlobalPkAddF16NoRtnInsts = false;
   bool HasAtomicBufferGlobalPkAddF16Insts = false;
+  bool HasAtomicCsubNoRtnInsts = false;
```

Comment:
Nit: I'd prefer to capitalise it like "CSub" throughout to match the existing hasAtomicCSub method.